anymail/backends/forwardemail.py

```
@@ -0,0 +1,192 @@
+        if emails:
+            # Nodemailer (Forward Email) accepts a comma-separated replyTo.
+            self.data["replyTo"] = ", ".join(
+                email.format(idna_encode=self.backend.idna_encode) for email in emails
+            )
+
+    def set_extra_headers(self, headers):
+        # Forward Email requires header values to be strings.
+        # Stringify ints and floats; anything else is the caller's responsibility.
+        for key, value in headers.items():
+            if isinstance(value, BASIC_NUMERIC_TYPES):
+                value = str(value)
```

Ruff (0.15.20)

[warning] `for` loop variable `value` overwritten by assignment target

(PLW2901)